src/EditorFeatures/CSharpTest/AddRequiredParentheses/AddRequiredParenthesesTests.cs

```diff
@@ -331,33 +319,7 @@ public async Task TestEqualityPrecedence2()
 {
     void M()
     {
-        int x = 1 + 2 == 2 $$+ 3;
-    }
-}", RequireOtherBinaryParenthesesForClarity);
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddRequiredParentheses)]
-        public async Task TestEqualityPrecedence3()
-        {
-            await TestMissingAsync(
-@"class C
-{
-    void M()
-    {
-        int x = 1 $$+ 2 == 2 + 3;
-    }
-}", RequireRelationalBinaryParenthesesForClarity);
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddRequiredParentheses)]
-        public async Task TestEqualityPrecedence4()
```

Comment:
Why so many tests deleted in this file? Are they redundant? If so, why?

Reply:
`TestEqualityPrecedence2` was an exact duplicate of `TestEqualityPrecedence1`

`TestEqualityPrecedence4` was an exact duplicate of `TestEqualityPrecedence3`

I removed 2 and 4, and renumbered 3 to 2 to fill the gap.